requirements.yml

```diff
@@ -106,6 +106,9 @@
 - src: git+https://github.com/mother-of-all-self-hosting/ansible-role-mongodb.git
   version: v6.0.6-0
   name: mongodb
+- src: git+https://github.com/mother-of-all-self-hosting/ansible-role-mosquitto.git
```

Comment:
The README for this role could be improved by:

- linking to the official upstream website (https://mosquitto.org/). It doesn't seem to be doing that right now
- fix the broken **official container image** link going to `https://hub.docker.com/_/mosquitto`. It should be `https://hub.docker.com/_/eclipse-mosquitto` instead.

----

I've also noticed a typo in the name of this variable: `mosquitto_allow_anonymus_access`

----

Also:

- the `mosquitto_container_http_port` variable seems to be defined but to not be used in the configuration file and in the systemd service file
- `mosquitto_container_http_port` is a bad name for Mosquitto's MQTT port. It's not HTTP. Perhaps `mosquitto_container_mqtt_port` would be a better name?
- `mosquitto_container_mqtt_host_bind_port` is hardcoded to `1883`, just like the other variable. It's better if they are kept in sync by defining on in terms of the other. `mosquitto_container_mqtt_host_bind_port: "{{ mosquitto_container_mqtt_port }}"` in `defaults/main.yml`
- users don't actually need to adjust the port inside the container, it's just `mosquitto_container_mqtt_host_bind_port` that's relevant.. So you may mention the `mosquitto_container_mqtt_host_bind_port` variable on the `mosquitto.md` documentation page somewhere


----

In `tasks/install.yml`, it's better if all container-related Ansible tasks are close together.

Right now, `community.docker.docker_network` is way at the top of the file, while `community.docker.docker_image` is toward the bottom.

To be consistent with the other roles, it's better to move container network creation toward the bottom as well.